app/models/order_status.rb

```diff
@@ -16,34 +16,29 @@ class OrderStatus < ActiveRecord::Base
   end
 
   scope :for_facility, ->(facility) { where(facility_id: [nil, facility.id]).order(:lft) }
-  scope :new_os, -> { where(name: "New").limit(1) }
-  scope :inprocess, -> { where(name: "In Process").limit(1) }
-  scope :canceled, -> { where(name: "Canceled").limit(1) }
-  scope :complete, -> { where(name: "Complete").limit(1) }
-  scope :reconciled, -> { where(name: "Reconciled").limit(1) }
 
   def self.new_status
-    new_os.first
+    find_by(name: "New")
   end
 
   def self.complete_status
```

Comment:
Yes, that's what I'm proposing (although `new` might need to stay `new_status` to avoid conflicting with the initialize method).

I'm not sure how much fork specific code we have to worry about. We could do something like this. I'm not sure we need to worry too much about the scopes going away, anywhere that used them should be calling `first` on it, which would error on an instance of `OrderStatus`.

```
def self.complete
  find_by(name: "Complete")
end

def self.complete_status
  warn "Deprecated complete_status called. Use complete instead. At #{caller(1..1)}"
  complete
end

`